plugins/lasersensor/src/LaserSensorDriver.cpp

```diff
@@ -18,6 +18,9 @@ using namespace boost::placeholders;
 using namespace yarp::dev;
 
 const std::string YarpLaserSensorScopedName = "sensorScopedName";
+namespace {
+    YARP_LOG_COMPONENT(GAZEBOLASER, "gazebo-yarp-plugins.plugins.GazeboYarpLaserSensor")
```

Comment:
I may be wrong, but this duplicates the definition in https://github.com/robotology/gazebo-yarp-plugins/pull/564/files#diff-db6c9bf936c1f5304516f6e46161f605ba89fd45c5f9db8d8ca905bbc078f14aR24, I guess just one and only one is necessary/required.

Reply:
I think I can put it in header only